src/components/header/_Header.scss

```diff
@@ -0,0 +1,49 @@
+@import url("https://fonts.googleapis.com/css2?family=Alfa+Slab+One&display=swap");
+
+/*Variables*/
+$font-logo: "Alfa Slab One", "Arial Black", cursive;
+$font-primry: "Manrope", Arial, sans-serif;
+$white: #ffffff;
+$black-one: #0000001f;
+$black-two: #00000029;
+
+/*Functions/Mixins*/
+@function calculateRem($size) {
+  $remSize: $size / 16px;
+  @return #{$remSize}rem;
+}
+
+.header {
+  width: 100%;
+  height: auto;
+  
+  &__legend {
+    color: $white;
+    font-family: $font-primry;
+    font-weight: 400;
+    font-size: calculateRem(24px);
+    text-align: center;
+    text-shadow: 0px 3px 6px $black-two;
```

Comment:
Please use `calculateRem` instead of px based measures

Reply:
Attended